ui/src/jobs/details/EnsemblingJobDetailsView.js

```diff
@@ -31,47 +28,44 @@ export const EnsemblingJobDetailsView = ({ projectId, jobId, ...props }) => {
   const hasInitiallyLoaded = useInitiallyLoaded(isLoaded);
 
   return (
-    <EuiPage>
-      <EuiPageBody>
-        {!hasInitiallyLoaded ? (
-          <EuiFlexGroup direction="row">
-            <EuiFlexItem grow={true}>
-              <EuiLoadingContent lines={3} />
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        ) : error ? (
-          <EuiCallOut
-            title="Sorry, there was an error"
-            color="danger"
-            iconType="alert">
-            <p>{error.message}</p>
-          </EuiCallOut>
-        ) : (
-          <Fragment>
-            <EuiPageHeader>
-              <EuiPageHeaderSection>
-                <PageTitle
-                  icon="sqlApp"
-                  iconSize="l"
-                  size="s"
-                  title={jobDetails.name}
-                  prepend={
-                    <StatusBadge
-                      status={JobStatus.fromValue(jobDetails.status)}
-                    />
-                  }
-                />
-              </EuiPageHeaderSection>
-            </EuiPageHeader>
-
+    <EuiPageTemplate restrictWidth="95%">
```

Comment:
I can imagine that the width restriction would be the same on all the app pages. Maybe we can parameterize it? Something similar to this: https://github.com/elastic/eui/blob/aea4705f832b27e253ce433ea0ccfcae7a7414d4/wiki/consuming.md#a-recommended-method-to-consume-theming-variables-using-emotion

Reply:
@romanwozniak are you suggesting consuming some Eui Global variables using Emotion as in the link? Or is it merely about the "parameterising" part which we currently do by defining configs? https://github.com/caraml-dev/turing/blob/main/ui/src/config.js#L51

Reply:
> Or is it merely about the "parameterising" part which we currently do by defining configs?
I can do this in the meantime and then see if we're unifying all these variables in `mlp-ui`